dts/xtensa/intel/intel_adsp_cavs25.dtsi

```diff
@@ -108,6 +109,7 @@
 			compatible = "intel,adsp-mem-window";
 			reg = <0x71a18 0x8>;
 			memory = <&sram0>;
+			read-only;
```

Comment:
Hmm, the readonly definitions don't match SOF now:
https://github.com/thesofproject/sof/blob/main/src/platform/intel/cavs/lib/mem_window.c#L32

For all cAVS platforms, we have windows 0 and 3 as read-only and I believe same should apply for ace15.

FYI to @softwarecki who made comments about the settings earlier.

Reply:
> FYI to @softwarecki who made comments about the settings earlier.

that was already addressed.